### PR TITLE
Feature/languages

### PR DESCRIPTION
**Pull request by AndrewC55**
This PR is to do this ticket [here](https://zube.io/group3-2/group3coursework/c/32).

I have also split the reports and objects into separate packages as well so that the workspace isn't so cluttered and made a handler with a switch statement in it

### CODE REVIEW

File: src/main/java/com/github/group3coursework/Reports/PeopleWhoSpeakDiffLang.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package com.github.group3coursework.Reports;
```
Comment (JudeScally) [inline src/main/java/com/github/group3coursework/Reports/PeopleWhoSpeakDiffLang.java]: Not major, but maybe this file could be named better. "LanguageReport" maybe

Comment (AndrewC55) [reply src/main/java/com/github/group3coursework/Reports/PeopleWhoSpeakDiffLang.java]: Do you mean the package? Or the 'PeopleWhoSpeakDiffLang'

Comment (JudeScally) [reply src/main/java/com/github/group3coursework/Reports/PeopleWhoSpeakDiffLang.java]: Yeah the 'PeopleWhoSpeakDiffLang'. I don't know how to do a full file comment :/

Comment (AndrewC55) [reply src/main/java/com/github/group3coursework/Reports/PeopleWhoSpeakDiffLang.java]: Aye you're right, name's confusing

**Review by JudeScally — APPROVED** [review]

**Pull request merged**